src/actions-types/PasswordActionsType.js

```diff
@@ -0,0 +1,18 @@
+export const FORGOT_PASSWORD_INITIALIZED = 'FORGOT_PASSWORD_INITIALIZED';
+export const FORGOT_PASSWORD_SUCCESS = 'FORGOT_PASSWORD_SUCCESS';
+export const FORGOT_PASSWORD_ERROR = 'FORGOT_PASSWORD_ERROR';
+export const RESET_PASSWORD_INITIALIZED = 'RESET_PASSWORD_INITIALIZED';
+export const RESET_PASSWORD_SUCCESS = 'RESET_PASSWORD_SUCCESS';
+export const RESET_PASSWORD_ERROR = 'RESET_PASSWORD_ERROR';
+
+export const forgotPasswordInitialized = () => ({ type: FORGOT_PASSWORD_INITIALIZED });
```

Comment:
@luc-tuyishime, why do you have these arrow functions that return the type of the action?

Reply:
Normally this way is for passing dynamic actions as parameters but I don't really need it so I have removed it already.